### PR TITLE
Simplify CID Deployment doc

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This step will create a new CUR for you. You can request a [backfill](https://do
 2. Click the **Launch CloudFormation button** below to open the **pre-populated stack template** in your CloudFormation console and select **Next**.
 
 	- [Launch CloudFormation Template](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?&templateURL=https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/cur-aggregation.yaml&stackName=CID-CUR-Destination&param_CreateCUR=True&param_DestinationAccountId=REPLACE%20WITH%20CURRENT%20ACCOUNT%20ID&param_SourceAccountIds=)
-	
+
 ![Images/multi-account/basic-setup-step1.png](/Cost/200_Cloud_Intelligence/Images/multi-account/basic-setup-step1.png?classes=lab_picture_small)
 
 3. Make sure CreateCUR set to **true**.
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:48]: 3. Make sure the CreateCUR parameter is set to **true**

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 2]
```diff
@@ -154,52 +154,56 @@ If you want to deploy **CUDOS**, **KPI** and **Cost Intelligence Dashboard** in
 
 {{%expand "Click here to continue with Advanced Setup" %}}
 
-
-
-## Deployment Overview
+## Architecture
 
 ![Images/arch5.png](/Cost/200_Cloud_Intelligence/Images/arch5.png?classes=lab_picture_small)
+The Cost and Usage report is generated in one or multiple Management (Payer) accounts. S3 replication transfer CUR files from **Source** (**Management Account**) to the **Destination** (**Data Collection Account**).
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:160]: The Cost and Usage report (CUR) is generated in one or multiple Management (Payer) accounts. S3 replication copies CUR files from **Source** (**Management Account**) to the **Destination** (**Data Collection Account**).

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 3]
```diff
@@ -154,52 +154,56 @@ If you want to deploy **CUDOS**, **KPI** and **Cost Intelligence Dashboard** in
...
 
-Advanced deployment supports multiple scenarios:
+This architecture support aggregation of CURs from multiple **Sources** to one **Data Collection Account**. This is useful if you want have a visibility across multiple Management (Payer) Accounts.
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:162]: This architecture supports aggregation of CURs from multiple **Sources** to one **Data Collection Account**. This is useful if you want have visibility across multiple Management (Payer) Accounts.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 4]
```diff
@@ -154,52 +154,56 @@ If you want to deploy **CUDOS**, **KPI** and **Cost Intelligence Dashboard** in
...
 
-Advanced deployment supports multiple scenarios:
+This architecture support aggregation of CURs from multiple **Sources** to one **Data Collection Account**. This is useful if you want have a visibility across multiple Management (Payer) Accounts. 
 
-1. Deploying Dashboards into an account other than the management (payer) account.
-2. Deploying Dashboards into an account, where multiple CURs, from multiple payer accounts are aggregated.
-3. Also Advanced deployment option is good for a situation where you do not have access to the management (payer) account, but want to deploy the Dashboaeds on top of multiple Linked Accounts.
-
-You will be deploying CloudFormation templates into two different types of places: your **Source Accounts** (typically management or payer account, but can be also a linked account as well) and your **Destination** or **Data Collection Account** (where you will deploy dashboards). You can create a new Linked Account or use an existing one. 
+Also the same aggregation can be used when you do not have access Management (Payer) and you need a visibility across multiple Linked Accounts that belong to a single Business Unit.
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:164]: Also the same aggregation can be used when you do not have access to the Management (Payer) account and you need visibility across multiple Linked Accounts that belong to a single Business Unit.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 5]
```diff
@@ -154,52 +154,56 @@ If you want to deploy **CUDOS**, **KPI** and **Cost Intelligence Dashboard** in
...
+## Deployment overview
 
 ![Images/arch6.png](/Cost/200_Cloud_Intelligence/Images/arch6.png?classes=lab_picture_verysmall)
 
-There will be 3 steps related to CloudFormation deployment: 
-1. In **Destination** or **Data Collection Account**, the CloudFormation template will create an S3 bucket for CUR aggregation `cid-{accountid}-shared`.
-2. In one or more **Source Accounts**. The CloudFormation template will create a new CUR, an S3 bucket, and a replication rule.
-3. In **Destination** or **Data Collection Account**, you will install another CloudFormation template that will create Dashboards.
+There are 3 major steps in the Deployment:
+* #1. Deploy a bucket for aggregated CUR in the **Data Collection Account**
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:171]: * #1. Deploy an S3 bucket for aggregated CUR in the **Data Collection Account**

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 6]
```diff
@@ -154,52 +154,56 @@ If you want to deploy **CUDOS**, **KPI** and **Cost Intelligence Dashboard** in
...
+* #2. Deploy CUR, bucket and a replication policy in **Source** Accounts (can be one or many Sources).
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:172]: Deploy CUR, S3 bucket and a replication policy in **Source** Accounts (can be one or many Sources).

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 7]
```diff
@@ -154,52 +154,56 @@ If you want to deploy **CUDOS**, **KPI** and **Cost Intelligence Dashboard** in
...
+* #3. Deploy Cloud Intelligence Dashboards (CID) Stack in **Data Collection Account**
+
+If your Management(payer) / Source  account is the same as your Destination account (where you want to deploy the dashboards) you can follow the steps for **Destination Account** only, and choose to activate Local CUR in the CFN parameter.
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:175]: and choose  "True" in CreateCUR parameter in the CFN template

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 8]
```diff
@@ -154,52 +154,56 @@ If you want to deploy **CUDOS**, **KPI** and **Cost Intelligence Dashboard** in
...
+## CUR Bucket Structure
+Each indivudal CUR has a prefix `cur/<account>` so the aggregated CUR has the following structure:
 
 ```html
 s3://<prefix>-<destination-accountid>-shared/
 	cur/<src-account1>/cid/cid/year=XXXX/month=YY/*.parquet
 	cur/<src-account2>/cid/cid/year=XXXX/month=YY/*.parquet
 	cur/<src-account3>/cid/cid/year=XXXX/month=YY/*.parquet
 ```
+This structure allows easily aggregate multiple CURs and allow a Glue crawler manage partitions: source_account_id, year, and month.
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:186]: This structure allows for easy aggregation of multiple CURs and allows a Glue crawler to manage partitions: source_account_id, year, and month.

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 9]
```diff
@@ -236,34 +244,14 @@ Here we will deploy the CFN template but setting the CFN parameters for a Destin
     ------------ | -------------
 
 8.  Once complete, the stack will show **CREATE_COMPLETE**.
-   **NOTE:** It takes 24 hours for your first CUR to be delivered
+
+**NOTE:** It takes 24 hours for your first CUR to be delivered. You can proceed with Deployment of dashboards, but they will be empty.
     ------------ | -------------
 
 9. It will take about 24 hours for your CUR to populate and replicate to your destination (data collection) account where you will deploy the dashboards. Return to this step after 24 hours. We also recommend creating a Support Case in Service=Billing and category=Invoices and Reporting, requesting a backfill of your CUR (name=cid) with 12 months of data. Case must be created from your Source Account (Management/Payer account).
```
Comment (shangop) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:251]: How did this become number 9. when it shows number 8 on the URL currently in place. Did you add a step?

**Comment by awsdunc** [comment]
@iakov-aws is this PR still in progress?

**Pull request merged**